components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsMessageFlow/src/EIAnalyticsMessageFlow.jsx

```diff
@@ -55,6 +55,9 @@ let svgCenter = {
 class EIAnalyticsMessageFlow extends Widget {
     constructor(props) {
         super(props);
+        this.props.glContainer.setTitle(
```

Comment:
This should be done with widget options. Please remove.
See: https://github.com/wso2/carbon-dashboards/pull/1042#issue-212959511